sherpa/utils/__init__.py

```diff
@@ -119,6 +120,7 @@
 
 
 class NoNewAttributesAfterInit(object):
+
```

Comment:
If you were to do anything, I'd suggest just adding a commit that removes these lines (as it seems to me to be a safer option). However, I am not sure it needs to be done in this PR (unless pep8/pyflakes/whatever) now complains about these lines: I only noticed them from the git diff view.